integration/internal/container/ops.go

```diff
@@ -255,3 +255,15 @@ func WithCDIDevices(cdiDeviceNames ...string) func(*TestContainerConfig) {
 		c.HostConfig.DeviceRequests = append(c.HostConfig.DeviceRequests, request)
 	}
 }
+
+func WithCapability(capability string) func(*TestContainerConfig) {
+	return func(c *TestContainerConfig) {
+		c.HostConfig.CapAdd = strslice.StrSlice([]string{capability})
+	}
+}
+
+func WithDropCapability(capability string) func(*TestContainerConfig) {
+	return func(c *TestContainerConfig) {
+		c.HostConfig.CapDrop = strslice.StrSlice([]string{capability})
+	}
+}
```

Comment:
These options should take a variable number of arguments and append to the config instead of overriding it, like `WithCDIDevices` above.

```suggestion
func WithCapabilities(caps ...string) func(*TestContainerConfig) {
	return func(c *TestContainerConfig) {
		c.HostConfig.CapAdd = append(c.HostConfig.CapAdd, strslice.StrSlice(caps))
	}
}

func WithDropCapabilities(caps ...string) func(*TestContainerConfig) {
	return func(c *TestContainerConfig) {
		c.HostConfig.CapDrop = append(c.HostConfig.CapDrop, strslice.StrSlice(caps))
	}
}
```